tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c

```diff
@@ -0,0 +1,525 @@
+        }
+    };
+    ng_ipv6_addr_t *out = NULL;
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_EQUAL_INT(TEST_UINT16, ng_ipv6_netif_find_addr(&out, &addr));
+    TEST_ASSERT_NOT_NULL(out);
+    TEST_ASSERT(out != &addr);
+    TEST_ASSERT_EQUAL_INT(true, ng_ipv6_addr_equal(out, &addr));
+}
+
+static void test_ipv6_netif_find_addr_local__no_iface(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+
+    TEST_ASSERT_NULL(ng_ipv6_netif_find_addr_local(TEST_UINT16, &addr));
+}
+
+static void test_ipv6_netif_find_addr_local__wrong_iface(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_NULL(ng_ipv6_netif_find_addr_local(TEST_UINT16 + 1, &addr));
+}
+
+static void test_ipv6_netif_find_addr_local__wrong_addr(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0xf0, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_NULL(ng_ipv6_netif_find_addr_local(TEST_UINT16, &addr));
+}
+
+static void test_ipv6_netif_find_addr_local__success(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+    ng_ipv6_addr_t *out = NULL;
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_NOT_NULL((out = ng_ipv6_netif_find_addr_local(TEST_UINT16, &addr)));
+    TEST_ASSERT(out != &addr);
+    TEST_ASSERT_EQUAL_INT(true, ng_ipv6_addr_equal(out, &addr));
+}
+
+static void test_ipv6_netif_match_prefix__success1(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0x01, 0x02, 0x02, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+    ng_ipv6_addr_t *out = NULL;
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_EQUAL_INT(TEST_UINT16, ng_ipv6_netif_match_prefix(&out, &addr));
+    TEST_ASSERT_NOT_NULL(out);
+    TEST_ASSERT(out != &addr);
+    TEST_ASSERT_EQUAL_INT(23, ng_ipv6_addr_match_prefix(out, &addr));
+}
+
+static void test_ipv6_netif_match_prefix__success2(void)
+{
+    ng_ipv6_addr_t addr = {{
+            0x01, 0x02, 0x33, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x00
+        }
+    };
+    ng_ipv6_addr_t *out = NULL;
+
+    test_ipv6_netif_add_addr__success();
+
+    TEST_ASSERT_EQUAL_INT(TEST_UINT16, ng_ipv6_netif_match_prefix(&out, &addr));
+    TEST_ASSERT_NOT_NULL(out);
+    TEST_ASSERT(out != &addr);
+    TEST_ASSERT_EQUAL_INT(18, ng_ipv6_addr_match_prefix(out, &addr));
+}
+
+static void test_ipv6_netif_match_prefix__success3(void)
```

Comment:
Yes, but currently there is only one. I may adapt them later on, as I did in #2613. ;-)